pom.xml

```diff
@@ -132,14 +143,14 @@
                 <goals>
                   <goal>sign</goal>
                 </goals>
+                <configuration>
```

Comment:
Also, I suspect we will need this:
https://github.com/sonatype-nexus-community/nexus-repository-r/blob/master/pom.xml#L106, if only because these two projects have the same parent, so likely will need the same property override for `pushChanges`.

Reply:
@bhamail did you want me to make this change right now, or try it without first?